GitHub-Test-3/jg-gitTest-3.md

```diff
@@ -0,0 +1,6 @@
+# The
+## Snow
+### Is
+#### Crazy
+yeah
```

Comment:
@julianjmg That's a good start wtih testing lots of different-sized headers in markdown! 
And *this* message that I am writing is called a "line comment" made directly on a  line of code in a commit to a remote GitHub repo. You'll be writing one of these on another student's commit + Pull Request for the last stage of the git/GitHub test.